tools/nvqpp/nvq++.in

```diff
@@ -321,6 +321,7 @@ CXX=${LLVMBIN}clang++${llvm_suffix}
 LINKER_CXX=${CXX}
 CMAKE_FALLBACK_HOST_CXX=${CXX}
 TARGET_ARGS=()
+HOST_TARGET=$(${LLVMBIN}llc${llvm_suffix} --version | grep "Default target" | head -1 | cut -d':' -f2 | tr -d ' ')
```

Comment:
Q: Does `llc` binary have a suffix? I thought only `clang` had one, for e.g. `clang-16`...
(I checked that this works, and `llvm_suffix` is empty, so its fine)

Reply:
All other references to `llc` in this script also include `${llvm_suffix}`, so I figured it would be most consistent to include it, but as you said, it is indeed empty in our standard configuration.